public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -74,7 +74,7 @@ function getStyles(theme: GrafanaTheme2) {
       text-align: start;
       line-break: anywhere;
       margin-top: -${theme.spacing(0.25)};
-      margin-right: ${theme.spacing(4)};
+      margin-right: ${theme.spacing(6)};
```

Comment:
Is this an accidental change? Or how is this related with the rest of the PR?

Reply:
This change was on purpose to make sure `LogContext` still looks great. Using a flex on the collapse bar so checked the components that use `Collapse` and `LogContext` looked like it could use a larger margin on the right.

Reply:
I see, thanks! How can I test this within your changes? Because using Loki in Explore there's nothing on the right of the query:

![imagen](https://github.com/grafana/grafana/assets/1069378/ccc18f00-9837-437a-bc85-753bb505372d)

Reply:
You should just be able to checkout this branch, build, run Loki and then open the `LogContext` as it's only adding extra margin right.

Doesn't look like you're seeing the revert icon in the far right (where the orange part is in your image). Is that missing from the DOM? (`Revert to initial log context query`)

![Screenshot 2023-07-04 at 12 57 50](https://github.com/grafana/grafana/assets/90795735/26aa9aa9-932e-4ab3-b1be-ce985c20c9b4)